tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_different_cves.yaml

```diff
@@ -0,0 +1,45 @@
+---
+# conf 1
+- tags:
+  - test_scan_different_cves
+  apply_to_modules:
+  - test_scan_different_cves
+  sections:
+  - section: vulnerability-detector
+    elements:
+    - enabled:
+        value: 'yes'
+    - run_on_start:
+        value: 'no'
```

Comment:
I think this setting is outdated because you have to wait 10 seconds for the feed import

Reply:
Ok, thanks, I will change it : )

Reply:
https://github.com/wazuh/wazuh-qa/pull/801/commits/8e51bbe88237dd5b71719ade7623c8250530e98a ty